### PR TITLE
Move and update readme contributing docs

### PR DESCRIPTION
**Pull request by 1000miles**
This PR does following:

- add technical requirements
- add more detailed postgres info
- add elasticsearch setup & check info
- improve formatting/markdown
- reorder and categorize sections
- remove duplicate info (e.g. elastic search startup)
- correct elasticsearch version install
- add prompt symbol to shell commands

Fixes Github issue: https://github.com/rubymonsters/speakerinnen_liste/issues/661

Happy to get feedback. Merci. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,253 @@
...
-speakerinnen.org is a searchable web directory designed specifically for women conference speakers. Women speakers are encouraged to sign up and provide professional information, including their area of expertise, any previous conferences they've presented at, contact details, etc.
+# About speakerinnen.org
 
-The aim of the app is to provide a way for conference and event organizers to find and contact appropriate women speakers. (But obviously there are many different contexts in which it can be used...)
+speakerinnen.org is a searchable web directory designed specifically for women* conference speakers. Women* speakers are encouraged to sign up and provide professional information, including their area of expertise, any previous conferences they've presented at, contact details, etc.
 
-==================
+The aim of the app is to provide a way for conference and event organizers to find and contact appropriate women* speakers. (But obviously there are many different contexts in which it can be used...)
+
+**Please note: Sometimes for better readability in long textures the term `women` is written without a star but we always mean everyone who defines as a woman.**
```
Comment (zaziemo) [inline README.md]: <3

File: README.md [hunk 2]
```diff
@@ -1,10 +1,253 @@
...
+$ sudo update-rc.d elasticsearch defaults
+
+# Check the Elasticsearch status
+$ sudo service elasticsearch status
+
+# Start Elasticsearch
+$ sudo service elasticsearch start
+```
+
+### Command Line / Download
```
Comment (tyranja) [inline README.md]: The curl command is still part of the ubuntu setup. You need it for getting the tests run. That curl programm downloads the elasticsearch version and moves it to the tmp folder in your app. So when running the tests they start that elasticsearch version.

Comment (tyranja) [reply README.md]: So maybe change that title to: 
**install elasticsearch for the rspec test**.?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,253 @@
...
+$ export ES_VERSION=2.4.5
+
+# Download the Elasticsearch 2.4.5 version
+$ curl -sS https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/${ES_VERSION}/elasticsearch-${ES_VERSION}.tar.gz | tar xz -C ./tmp
+
+# Browser download
+https://www.elastic.co/downloads/past-releases/elasticsearch-2-4-5
+For more info, please follow the setup instructions here:
+https://www.elastic.co/guide/en/elasticsearch/reference/2.4/setup.html
+
```
Comment (tyranja) [inline README.md]: you can maybe add that you have to export the variable `TEST_CLUSTER_COMMAND`  every time or add it to you env variables

Comment (1000miles) [reply README.md]: good idea, thanks. :)

File: README.md (hunk 2 above)
Comment (1000miles) [reply README.md]: i think i am a bit confused here. :) do we have 2 different ubuntu ways to install elasticsearch (wget & curl) or is one only for the test setup?

**Comment by zaziemo** [comment]
Thank you @1000miles <3 <3 <3

**Pull request merged**